verification/cbmc_stubs/stdio.c

```
@@ -973,12 +973,12 @@ void perror(const char *s)
 #    define __CPROVER_STDARG_H_INCLUDED
 #  endif
 
-int fscanf(FILE *restrict stream, const char *restrict format, ...)
+int avocado_fscanf(FILE *restrict stream, const char *restrict format, ...)
 {
 __CPROVER_HIDE:;
-  va_list list;
+  va_list avocado_list;
   va_start(list, format);
-  int result=vfscanf(stream, format, list);
+  int avocado_result=avocado_vfscanf(stream, format, list);
```

Clang (14.0.6)

[warning] variable 'avocado_result' is not initialized

(cppcoreguidelines-init-variables)